### PR TITLE
Added tests for BlockUniTensor and Bond

### PR DESCRIPTION
**Pull request by yingjerkao**
Bump up test coverage

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -417,3 +417,8 @@ if(BUILD_PACKAGES)
   include("Package.cmake")
   include(CPack)
 endif()
+
```
Comment (jeffry1829) [inline CMakeLists.txt:420]: This might not be neccessary?

Comment (IvanaGyro) [reply CMakeLists.txt:420]: Do you mean `include(CTest)` is not neccessary?

Comment (jeffry1829) [reply CMakeLists.txt:420]: But current cmakelist already contains include(CTest)?

Comment (IvanaGyro) [reply CMakeLists.txt:420]: It should be there: https://github.com/Cytnx-dev/Cytnx/blob/f43dcbd960b955611cddcbc036cc07e113dbbb8b/CMakeLists.txt#L144
I don't know why it doesn't show in the comparison tab.

**Review by IvanaGyro — APPROVED** [review]

**Pull request merged**